clang/include/clang/StaticAnalyzer/Core/PathSensitive/SVals.h

```diff
@@ -326,6 +326,12 @@ class LocAsInteger : public NonLoc {
   static bool classof(SVal V) { return V.getKind() == LocAsIntegerKind; }
 };
 
+/// The simplest example of a concrete compound value is nonloc::CompoundVal,
+/// which represents a concrete r-value of an initializer-list or a string.
+/// Internally, it contains an llvm::ImmutableList of SVal's stored inside the
+/// literal.
+///
+/// Source: https://github.com/haoNoQ/clang-analyzer-guide, v0.1, section 5.3.2.
```

Comment:
If we want to phase out the guide, I don't think it's valuable to mention it here.
If you insist, let's mention this additional resource in the dev docs, but not directly here.

Reply:
Yeah please nuke this thank you ^.^

Sometimes we cite the academic papers that describe the specific approaches we've implemented (IPA is a good example) but this one's definitely inside out. The source code should be the primary source of truth about the source code.